backend/src/graphql/resolver/CommunityResolver.test.ts

```diff
@@ -48,6 +48,9 @@ beforeAll(async () => {
   query = testEnv.query
   con = testEnv.con
   await cleanDB()
```

Comment:
Does it make sense to insert in cleanDB the reset id auto increment for all entities/tables than to explicitly clear only a few ones afterwards?